room/runtime/src/main/java/androidx/room/DatabaseConfiguration.java

```diff
@@ -61,6 +61,9 @@
     @Nullable
     public final List<RoomDatabase.Callback> callbacks;
 
+    @Nullable
+    public final List<RoomDatabase.PrepackagedCallback> prepackagedCallbacks;
```

Comment:
I'm thinking it might be best / simpler if instead of a list we only take one callback. The reason is we only allow one of the configurations `createFromFile` / `createFromAsset` / `createFromStream`, and since the callback is passed on those builder methods it seems weird that calling it again to change the configuration (to like update the file name) you end up passing the same callback and it'll get invoked twice.